lib/Echidna/Processor.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Echidna.Processor where
+
+import Control.Arrow              (second)
+import Control.Lens
+import Control.Monad.IO.Class     (MonadIO(..))
+import Control.Exception          (Exception)
+import Control.Monad.Catch        (MonadThrow(..))
+import Data.Aeson                 (decode, Value(..))
+import System.Directory           (findExecutable)
+import System.Process             (StdStream(..), readCreateProcessWithExitCode, proc, std_err)
+import System.Exit                (ExitCode(..))
+import GHC.Word                   (Word32)
+
+import qualified Data.ByteString.Lazy.Char8 as BSL
+import qualified Data.HashMap.Strict as M
+import qualified Data.Text           as T
```

Comment:
we can import the `Text` type so it doesn't have to be qualified elsewhere

Reply:
Fixed.